### PR TITLE
chore: prepare for ruff-format

### PR DESCRIPTION
**Pull request by henryiii**
Prepare for ruff-format.

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -21,12 +21,6 @@ repos:
   - id: name-tests-test
     args: ["--pytest-test-first"]
 
-- repo: https://github.com/asottile/setup-cfg-fmt
-  rev: v2.5.0
-  hooks:
-  - id: setup-cfg-fmt
-    args: [--include-version-classifiers, --max-py-version=3.11]
-
```
Comment (agoose77) [inline .pre-commit-config.yaml:29]: Thanks!

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -21,12 +21,6 @@ repos:
...
 - repo: https://github.com/psf/black-pre-commit-mirror
   rev: 23.10.0
   hooks:
```
Comment (agoose77) [inline .pre-commit-config.yaml:26]: Does `ruff-format` not replace `black`? Or is this just the prep-PR?

Comment (henryiii) [reply .pre-commit-config.yaml:26]: Prep. ;)

**Comment by agoose77** [comment]
LGTM. Let me know if I'm OK to merge.

**Comment by henryiii** [comment]
Sure. :)

**Pull request merged**